needrestart

```diff
@@ -1265,6 +1286,25 @@ if(defined($opt_l) && !$uid) {
 	    }
 	}
     }
+
+	## GUESTS
+	$ui->vspace();
+	if (! @guests) {
+		$ui->notice(__ 'No VM guests are running outdated hypervisor (qemu) binaries on this host.') unless($opt_b || $opt_m eq 'e');
+	}
+	else {
+		if($opt_m eq 'e') {
+			push(@easy_hints, __ 'outdated VM guests');
+		}
+		else {
+			unless($opt_p || $opt_b) {
+				$ui->notice(__ 'VM guests are running outdated hypervisor (qemu) binaries on this host:');
+				foreach ( @guests ) {
+				$ui->notice(__ " $_");
```

Comment:
Sorry, I might not be perl affine enough to get your point on this :-/

You mentioned you want to no localize it here but later - does that mean localization like in translation?
If so I do not see what I should move down from here.
Do you have a pointer or suggestion what it would look like that you request?

Reply:
I pushed the fix to this for the PR and resolve this thread, for the one below you'd need to let me know what you want/need.